tools/ui-generator/presto-sample/package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "presto-sample",
+  "version": "0.0.0",
+  "private": false,
+  "scripts": {
+    "compile": "webpack --progress --colors --optimize-minimize",
+    "start": "node ./node_modules/webpack-dev-server/bin/webpack-dev-server.js --content-base dist/"
+  },
+  "dependencies": {
+    "axios": "^0.16.2",
+    "qr-image": "^3.2.0",
```

Comment:
Is this needed?

Reply:
Nope, though web polyfill needs this (carried over from pregel), it has been cleaned in the perf version.

Reply:
Mystique requires this, https://github.com/juspay/presto-ui/blob/master/core/src/WEB/JBridgeInterface.js#L29.

Will send out another PR to cleanup presto-ui/core